lib/str.ts

```diff
@@ -109,31 +112,31 @@ const Str = {
     /**
      * Escape text while preventing any sort of double escape, so 'X & Y' -> 'X &amp; Y' and 'X &amp; Y' -> 'X &amp; Y'
      *
-     * @param {String} s the string to escape
-     * @return {String} the escaped string
+     * @param s the string to escape
+     * @return string the escaped string
```

Comment:
Let's be consistent and start descriptions after variable names with capital letter in all functions
```suggestion
     * @param s The string to escape
     * @returns the escaped string
```